cv-basics/Playing_with_matrices/Playing_with_matrices.ipynb

```diff
@@ -0,0 +1,857 @@
+{
```

Comment:
<p>I guess index slicing is not done properly, 640 is out of image dimensions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Resolved the issue. Made the value dynamic, equating to half of the dimension of the image.